### ISSUE TITLE
Crash with non-text files

### ISSUE THREAD

**Issue by Tilix4** [douglaskastle/blender-addon-tester#30]
In the function `clean_file` if there is a non-text file in the targeted directory, I get a `UnicodeDecodeError: 'utf-8' codec can't decode byte 0x80 in position 3131: invalid start byte` (in my case it's for a `.DS_Store` OSX file). I also had the issue with image files.

I don't get what this function is supposed to do and if this issue is actually critical.

**Comment by douglaskastle** [comment]
I don't understand this.  What targeted directory are we talking about here can you show me an example?

**Comment by Tilix4** [comment]
The targeted directory is an addon directory, in OSX, the OS creates a `.DS_Store` binary file. Maybe the issue could be the same with a binary file, like an executable.

**Comment by douglaskastle** [comment]
Are you saying there is a file in the addon directory, .DS_Store, that is a binary file, and it causes a crash?

I don't have access to a osx machine, I'll try and replicate on linux.

**Comment by Tilix4** [comment]
Exactly. Maybe you can reproduce it with any compiled file or a picture.

**Comment by semagnum** [comment]
I got this as well - if I have a non-Unicode or binary file in the addon and the tester tries to read it, it crashes due to the Unicode error. My case is an *.odp file, but there's been leftover __pycache__'s that don't get deleted and also crash.

**Comment by Tilix4** [comment]
I modified the code to skip this check. Maybe we could have an option to skip it.
